config/jobs/kubernetes-sigs/structured-merge-diff/structured-merge-diff-presubmits.yaml

```diff
@@ -36,3 +36,21 @@ presubmits:
       testgrid-dashboards: sig-api-machinery-structured-merge-diff
       testgrid-tab-name: pr-gofmt
       testgrid-num-columns-recent: '30'
+  - name: pull-structured-merge-diff-benchmark
+    always_run: false
+    skip_report: true
+    decorate: true
+    path_alias: sigs.k8s.io/structured-merge-diff
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/benchmarkjunit:latest
+        command:
+        - /app/pkg/benchmarkjunit/app.binary
```

Comment:
nit: You can just use `/benchmarkjunit` if you include something like this in the `prow_image` bazel target: https://github.com/kubernetes/test-infra/blob/fd45432efbe1eecb848d9a87127a61db4f2eeee3/prow/cmd/branchprotector/BUILD.bazel#L19-L20

Reply:
Done -- built the binary to `/benchmarkjunit` and set as the `command`